src/nameserver/name_server_impl.cc

```diff
@@ -5572,8 +5572,12 @@ void NameServerImpl::OnLocked() {
         CreateSystemTableOrExit(SystemTableType::kGlobalVariable);
         InitGlobalVarTable();
     }
-    if (FLAGS_system_table_replica_num > 0 && db_table_info_[INFORMATION_SCHEMA_DB].count(DEPLOY_RESPONSE_TIME) == 0) {
-        CreateSystemTableOrExit(SystemTableType::kDeployResponseTime);
+    if (FLAGS_system_table_replica_num > 0) {
+        if (db_table_info_[INFORMATION_SCHEMA_DB].count(DEPLOY_RESPONSE_TIME) == 0) {
+            CreateSystemTableOrExit(SystemTableType::kDeployResponseTime);
```

Comment:
`..OrExist` already handles if table exist. My concern is the user upgrade from old version and `deploy_response_time` already exists with the `(string, string,  int, string)` schema, we won't correct the schema. So at least, we should detect the old table, and print a warning log about the potential count overflow